server/main.go

```diff
@@ -42,7 +48,8 @@ func main() {
 		},
 	}
 	cmd.Flags().StringVarP(&addr, "addr", "", addr, "addr to listen")
-	cmd.Flags().StringVarP(&upstream, "upstream", "", upstream, "specific the upstream mirror")
+	cmd.Flags().StringVarP(&keyDir, "keys", "", keyDir, "specify the directory where stores the private keys")
```

Comment:
The argument could just be `--key-dir` instead of `--keys`, which implies it accepts multiple key file paths rather than a folder contains them.